File: PRAKTIKUM MODUL 7/SOAL 5/PRAK605-2210817210008-Bima Sanjaya.c
#include <stdio.h>
int main()
{
    int MA[10][10],MB[10][10],MAXB[10][10];
    int a,b,c,d,hasil=0;
    scanf("%d",&d);
    printf("Matriks A\n");
    for(a=0;a<d;a++){
        for(b=0;b<d;b++){
            scanf("%d",&MA[a][b]);}}
    printf("Matriks B\n");
    for(a=0;a<d;a++){
        for(b=0;b<d;b++){
            scanf("%d",&MB[a][b]);}}
    for(a=0;a<d;a++){
        for(b=0;b<d;b++){
            for(c=0;c<d;c++){
                hasil=hasil+MA[a][c]*MB[c][b];}
            MAXB[a][b]=hasil;
            hasil=0;}}
    printf("Matriks AXB\n");
    for(a=0;a<d;a++){
        for(b=0;b<d;b++){
            printf("%d ",&MAXB[a][b]);}
        printf("\n");}
}

File: PRAKTIKUM MODUL 7/SOAL 5/PRAK605-2210817210008-Bima Sanjaya.py
a=int(input())
MA=[]
print('Matriks A')
for i in range (a):
    k=list(map(int,input().split()))
    MA.append(k)
MB=[]
print('Matriks B')
for i in range (a):
    k=list(map(int,input().split()))
    MB.append(k)
MAXB=[]
print('Matriks AXB')
for i in range (a):
    MAXB.append([])
    for j in range (a):
        y=0
        for k in range (a):
            y=y+MA[i][k]*MB[k][j]
        MAXB[i].append(y)
for baris in MAXB:
    for k in baris:
        print(k,end=' ')
    print()